scripts/rs-init.sh

```diff
@@ -1,37 +1,39 @@
 #!/bin/bash
 
-MONGO_NODES=(mongo1t:27027 mongo2t:27028 mongo3t:27029)
+MONGO_NODES_ARRAY=()
 
 echo "Waiting for mongo nodes serverStatus..."
-for mongo_node in "${MONGO_NODES[@]}"
+for mongo_node in $MONGO_NODES
 do
+  echo $mongo_node
```

Comment:
This `echo $mongo_node` can still be handy, since it'll run a curl command targeting it, could you further enhance the echo msg here, something like `echo "It'll check serverStatus uptime of mongo node $mongo_node ..."`